dev-libs/libstrophe/libstrophe-0.9.2.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
```

Comment:
This eclass isn't being used in this ebuild.

Reply:
It worked without, but I thought it depended on autotools, and I thought this automatically added that as a dependency?

Or is that an implicit system dependency?

Reply:
> 
> Or is that an implicit system dependency?

Yes :) It's not a big thing and not really an error in any way, just keeping the ebuild cleaner.

Reply:
Great! It didn't look like an implicit system dependency to me, but I'm glad that it is! I'll remove that then :).